docs/concepts/executor/serve.md

```diff
@@ -176,6 +176,47 @@ WorkerRuntime@ 1[L]: Executor CLIPTextEncoder started
 Just like that, our Executor is up and running.
 
 (kubernetes-executor)=
+## Serve from Deployment YAML
+In case you want a clear separation between deployment configuration and Executor logic, you can define those 
+configuration in a `Deployment` YAML configuration.
+This is an example `deployment.yml` config file:
```

Comment:
```suggestion
```

Reply:
I need to show the file name to the user, needed later

Reply:
OK, keep it. Can remove the "For example:" string